src/mysky/utils.ts

```diff
@@ -15,6 +16,26 @@ export async function getFullDomainUrl(this: SkynetClient, domain: string): Prom
   return getFullDomainUrlForPortal(portalUrl, domain);
 }
 
+// TODO: unit test
```

Comment:
probably not something we want to follow up with but rather implement in this PR?

Reply:
The unit tests have been implemented, will remove the comment.